proposals/0008-swift-prometheus.md

```diff
@@ -0,0 +1,150 @@
+# SwiftPrometheus - Prometheus Metrics in Swift
+
+* Proposal: SSWG-0008
+* Authors: [Jari Koopman / LotU](https://github.com/MrLotU)
+* Review Manager: TBD
+* Status:  [Implemented ](https://github.com/MrLotU/SwiftPrometheus/tree/MetricsLib)
+* Pitch: [Pitches/Prometheus](https://forums.swift.org/t/client-side-prometheus-implementation/18098/)
+
+## Package Description
+Prometheus client side implementation.
+
+|  |  |
+|--|--|
+| **Package Name** | `SwiftPrometheus` |
+| **Module Name** | `Prometheus` & `PrometheusMetrics` |
+| **Proposed Maturity Level** | [Sandbox](https://github.com/swift-server/sswg/blob/master/process/incubation.md#process-diagram) |
+| **License** | [Apache 2.0](https://github.com/MrLotU/SwiftPrometheus/blob/master/LICENSE) |
+| **Dependencies** | swift-nio 1.0.0..<3.0.0 (1.x.x & 2.x.x) - swift-metrics > 1.0.0 |
```

Comment:
does is actually support both swift-nio 1 an swift-nio 2?

Reply:
The idea here (also see https://github.com/MrLotU/SwiftPrometheus/pull/12) is to have a 1.x release for NIO 2 and a 0.x release for NIO 1